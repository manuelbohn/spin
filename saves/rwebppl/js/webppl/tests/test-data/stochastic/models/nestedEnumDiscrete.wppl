var marginal = Infer({model() {
  var x = flip();
  var y = flip();
  factor((x | y) ? 0 : -Infinity);
  return x;
}});

var model = function() {
  var x = sample(marginal);
  var y = sample(marginal);
  factor((x | y) ? 0 : -Infinity);
  return x;
};
