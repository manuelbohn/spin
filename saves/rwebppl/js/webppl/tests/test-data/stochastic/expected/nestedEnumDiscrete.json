{
  "hist": {
    "true": 0.75,
    "false": 0.25
  }
}
